Headers/NSObject+Blindside.m

```diff
@@ -23,4 +32,34 @@ - (void)bsAwakeFromPropertyInjection {
     
 }
 
+static const char * const kInitName = "init";
+static const size_t kInitLength = 4;
+
++ (SEL)bsOverriddenInitializer {
```

Comment:
I don't really like having this right here. If #39 gets merged first, I'd probably move this to `BSUtils` which is introduced by that patch set.